ci/citest/acceptance-test/multibox/ind-steps/step-vdc-images/init.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
-for img in "${IMAGES[@]}" ; do
-    item=$(grep "$img" <<< "$imgIndex")
-    IFS=';' read -a parts <<< "$item"
-    IFS=';' eval 'folders=($item)'
-    imgRemoteDir="${parts[-1]}"
+case "${image_type}" in
+    qemu) # stores directly in cache until we can use compressed files
+        IMAGES=( "${QEMU_IMAGES[@]}" )
+        LOCAL_STORAGE_PATH="${NODE_DIR}/guestroot/var/cache/qemu"
+        ;;
+    lxc) # stores on local image server
+        IMAGES=( "${LXC_IMAGES[@]}" )
+        LOCAL_STORAGE_PATH="${NODE_DIR}/guestroot/var/www/html/images"
+        ;;
+    *)
+        echo "Unknown image type."
```

Comment:
This should include `${image_type}` to make it a lot easier to debug. Also maybe we should fail when there's an unknown image type?

I figure this code should path ideally never be executed so if we fail more explicitly here, it'll make it easier to debug. An error message is easily missed in the log and it is likely to cause problems further down the line.